README.md

```diff
@@ -260,14 +307,55 @@ julia> f1(n) = @view A[:, 1];
 
 julia> f2(n) = @view A[:, 2];
 
-julia> x = readsas("productsales.sas7bdat", include_columns=[:ACTUAL,:PREDICT], 
-              number_array_fn=Dict(:ACTUAL => f1, :PREDICT => f2));
-Read productsales.sas7bdat with size 1440 x 2 in 0.32306 seconds
+julia> readsas("productsales.sas7bdat", include_columns=[:ACTUAL,:PREDICT], 
+               number_array_fn=Dict(:ACTUAL => f1, :PREDICT => f2));
+Read productsales.sas7bdat with size 1440 x 2 in 0.00041 seconds
+
+julia> A[1:5,:]
+5×2 Array{Float64,2}:
+ 925.0  850.0
+ 999.0  297.0
+ 608.0  846.0
+ 642.0  533.0
+ 656.0  646.0
+```
+
+### Metadata
+
+You may obtain meta data for a SAS data file using the `metadata` function.
+
+```julia
+julia> md = metadata("productsales.sas7bdat")
+SASLib.Metadata("productsales.sas7bdat", "US-ASCII", :LittleEndian, :none, 8192, 18, 1440, 10, Pair{Symbol,DataType}[:ACTUAL=>Float64, :PREDICT=>Float64, :COUNTRY=>String, :REGION=>String, :DIVISION=>String, :PRODTYPE=>String, :PRODUCT=>String, :QUARTER=>Float64, :YEAR=>Float64, :MONTH=>Float64])
+```
+
+It's OK to access the fields directly.
+```julia
+julia> fieldnames(md)
+9-element Array{Symbol,1}:
+ :filename   
+ :encoding   
+ :endianness 
+ :compression
+ :pagesize   
+ :npages     
+ :nrows      
+ :ncols      
+ :columnsinfo
+```
+
+If you already have a `SASLib.Handler` object from the `SASLib.open` call, you can get metadata directly from the handler:
+
+```julia
+julia> metadata(handler)
+SASLib.Metadata("productsales.sas7bdat", "US-ASCII", :LittleEndian, :none, 8192, 18, 1440, 10, Pair{Symbol,DataType}[:ACTUAL=>Float64, :PREDICT=>Float64, :COUNTRY=>String, :REGION=>String, :DIVISION=>String, :PRODTYPE=>String, :PRODUCT=>String, :QUARTER=>Float64, :YEAR=>Float64, :MONTH=>Float64])
 ```
 
 ## Why another package?
```

Comment:
Honestly, this package has matured quite a bit and keeps getting better. Its functionality is moving beyond just a port of some Python code. At this point, you might not need this section anymore. This isn't just a package you made for your own purposes, this is becoming the best way to use SAS files in Julia. I'd suggest merging this section and the one below it into 1 section titled Notes and just have 1 sentence that says something to the effect of "The core functionality of this package borrowed inspiration from Pandas' `read_sas` and `ReadStat.jl`."

Reply:
Agree 💯